frontend/assets/icons/wallets/metamask.svg

```diff
@@ -1 +1,44 @@
+<svg version="1.1" id="Layer_1" xmlns:ev="http://www.w3.org/2001/xml-events"
```

Comment:
Why was the change of this svg necessary? If you need to keep the new version, please remove `xmlns:xlink`, `xmlns:ev`, `xml:space` and other non-html compatible or [deprecated](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xml:space) html arguments

svgo is a useful tool for this 
 - https://marketplace.visualstudio.com/items?itemName=1000ch.svgo
 - https://jakearchibald.github.io/svgomg/

Reply:
I'm not sure what you mean, production always showed icons for me:

<img width="1552" alt="Screenshot 2022-11-30 at 19 56 52" src="https://user-images.githubusercontent.com/3393626/204884711-ef1475c4-03f1-4466-b891-ca3288869b2b.png">

Which browser you're using?